src/models/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './icons';
 export * from './i18n';
 export * from './iconConfiguration';
 export * from './advancedWorkspaceConfiguration';
+export * from './i18n';
```

Comment:
Is the presence of both exports of the same file normal?

Reply:
hmm I may have overlooked that.. fixing it now. thanks @HelloEdit